packages/services/api/src/modules/audit-logs/module.graphql.ts

```diff
@@ -0,0 +1,236 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  interface AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+  }
+
+  type AuditLogUserRecord {
```

Comment:
suggestion: 
```graphql
interface AuditLog {
    id: ID!
    eventTime: DateTime!
    record: AuditLogIdRecord!
}
  
type AuditLogIdRecord {
    userId: ID!
    userEmail: String!
    organizationId: ID!
    user: User # this one is nullable because it can be deleted!
    organization: Organization # this one is nullable because it can be deleted!
  }
```